### PR TITLE
docs: update documentation on new prometheus metrics

### PR DESCRIPTION
**Pull request by nerdeveloper**

### CODE REVIEW

File: content/docs/prometheus-metrics/_index.md [hunk 1]
```diff
@@ -6,9 +6,9 @@ weight: 7
 draft: false
 ---
 
-ChartMuseum exposes its [Prometheus metrics](https://prometheus.io/docs/concepts/metric_types/) at the `/metrics` route on the main port. This can be disabled with the `--disable-metrics` command-line flag or the `DISABLE_METRICS` environment variable.
+ChartMuseum exposes its [Prometheus metrics](https://prometheus.io/docs/concepts/metric_types/) at the `/metrics` route on the main port. This can be enabled with the `--enable-metrics` command-line flag or the `ENABLE_METRICS` environment variable.
 
-> Note that the Kubernetes chart currently disables metrics by default (`DISABLE_METRICS=true` is set in the chart).
+> Note that the Kubernetes chart currently disables metrics by default (`ENABLE_METRICS=false` is set in the chart). The `--disable-metrics` command-line flag has be deprecated and will only be available in `v0.14.0` and prior.
```
Comment (cbuto) [inline content/docs/prometheus-metrics/_index.md]: We should mention that it’s disabled by default in general and not only disabled in the chart.

**Review by cbuto — COMMENTED** [review]
Looks good! 🎉 

We can also update https://github.com/chartmuseum/www/blob/main/content/docs/how-to-run/all-cli-options.md to reflect the changes to the flags. 

We’ll want to wait for the 0.15.0 release to merge this.

File: content/docs/prometheus-metrics/_index.md (hunk 1 above)
Comment (nerdeveloper) [reply content/docs/prometheus-metrics/_index.md]: okay will update it now

**Comment by nerdeveloper** [comment]
> Looks good! 🎉
> 
> We can also update https://github.com/chartmuseum/www/blob/main/content/docs/how-to-run/all-cli-options.md to reflect the changes to the flags.
> 
> We’ll want to wait for the 0.15.0 release to merge this.

i will do that now

File: content/docs/prometheus-metrics/_index.md [hunk 2]
```diff
@@ -6,9 +6,9 @@ weight: 7
 draft: false
 ---
 
-ChartMuseum exposes its [Prometheus metrics](https://prometheus.io/docs/concepts/metric_types/) at the `/metrics` route on the main port. This can be disabled with the `--disable-metrics` command-line flag or the `DISABLE_METRICS` environment variable.
+ChartMuseum exposes its [Prometheus metrics](https://prometheus.io/docs/concepts/metric_types/) at the `/metrics` route on the main port. This can be enabled with the `--enable-metrics` command-line flag or the `ENABLE_METRICS` environment variable.
 
-> Note that the Kubernetes chart currently disables metrics by default (`DISABLE_METRICS=true` is set in the chart).
+> Note that the metric is disabled by default and on the Kubernetes chart (`ENABLE_METRICS=false` is set in the chart). The `--disable-metrics` command-line flag has be deprecated and will only be available in `v0.14.0` and prior.
```
Comment (cbuto) [inline content/docs/prometheus-metrics/_index.md]:
```suggestion
> Note that metrics are disabled by default (this includes the Kubernetes Helm chart). The `--disable-metrics` command-line flag has been deprecated and will only be available in `v0.14.0` and prior.
```

**Comment by nerdeveloper** [comment]
This PR is done and ready to be merged whenever it's time @jdolitsky @cbuto

**Pull request merged**